webapp/src/main/java/com/box/l10n/mojito/service/tm/search/TextUnitDTONativeObjectMapper.java

```diff
@@ -40,15 +41,18 @@ public TextUnitDTO mapObject(CriteriaResult cr) {
     // implementation to understand why getBoolean doesn't work. This
     // seems to work fine, use this code for now.
     t.setIncludedInLocalizedFile(Boolean.valueOf(includedInLocalizedFile));
-    t.setCreatedDate(new DateTime(cr.getDate(idx++)));
+    t.setCreatedDate(DateTimeUtils.fromDateToZonedDate(cr.getDate(idx++)));
     String assetDeleted = cr.getString(idx++);
     t.setAssetDeleted(Boolean.valueOf(assetDeleted));
     t.setPluralForm(cr.getString(idx++));
     t.setPluralFormOther(cr.getString(idx++));
     t.setRepositoryName(cr.getString(idx++));
     t.setAssetPath(cr.getString(idx++));
     t.setAssetTextUnitId(cr.getLong(idx++));
-    t.setTmTextUnitCreatedDate(new DateTime(cr.getDate(idx++)));
+    Date tmTextUnitCreatedDate = cr.getDate(idx++);
```

Comment:
please update all the relevant callsite with the function you've introduced, eg. here: fromDateToZonedDate